veg2hab/criteria.py

```diff
@@ -312,7 +321,13 @@ def evaluation(self) -> MaybeBoolean:
         return ~self.sub_criterium.evaluation
 
     def __str__(self):
-        return f"niet {self.sub_criterium}"
+        # Hier veranderen we "niet FGR is Duinen (F)" naar "niet FGR is Duinen (T)",
+        # want niet false == true
+        if type(self.sub_criterium) in [FGRCriterium, BodemCriterium, LBKCriterium]:
```

Comment:
`isinstance(self.sub_criterium, (FGRCriterium, BodemCriterium, LBKCriterium))`

Bovendien kan deze hele check toch weg, als we ervoor zorgen dat elk BeperkendCriterium een get_format_string functie heeft.